Core/tests/ExpressionCompletionFinder.cpp

```diff
@@ -55,8 +55,8 @@ TEST_CASE("ExpressionCompletionFinder", "[common][events]") {
     SECTION("Object or expression completions when type is string") {
       // clang-format off
       std::vector<gd::String> expectedCompletions{
-          "{ 0, string, 1, no prefix, MyObject, no object name, no behavior name, non-exact, not last parameter, no parameter metadata, with object configuration }",
-          "{ 3, no type, 1, no prefix, myVariable, no object name, no behavior name, non-exact, not last parameter, no parameter metadata, no object configuration }",
+          "{ 0, string, 2, no prefix, MyObject, no object name, no behavior name, non-exact, not last parameter, no parameter metadata, with object configuration }",
+          "{ 3, no type, 2, no prefix, myVariable, no object name, no behavior name, non-exact, not last parameter, no parameter metadata, no object configuration }",
```

Comment:
Variable type value of all tests are shifted because I added `gd::Variable::Type::Unknown`.